src/Event/ValueObjects/SubEventUpdate.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Event\ValueObjects;
+
+use CultuurNet\UDB3\Offer\ValueObjects\BookingAvailability;
+
+final class SubEventUpdate
+{
+    private int $subEventIndex;
+
+    private ?Status $status = null;
+
+    private ?BookingAvailability $bookingAvailability = null;
+
+    public function __construct(int $subEventIndex)
+    {
+        $this->subEventIndex = $subEventIndex;
+    }
+
+    public function withStatus(Status $status): SubEventUpdate
+    {
+        $clone = clone $this;
+        $clone->status = $status;
+        return $clone;
+    }
+
+    public function withBookingAvailability(BookingAvailability $bookingAvailability): SubEventUpdate
+    {
+        $clone = clone $this;
+        $clone->bookingAvailability = $bookingAvailability;
+        return $clone;
+    }
+
+    public function getSubEventIndex(): int
```

Comment:
Nit: Maybe this would better be named `getSubEventId()` because it's called `id` in the JSON of the update request, and there's also a ticket in consideration to add it to the JSON of the complete event details: https://jira.uitdatabank.be/browse/III-4126
I know that it's actually just the index in the list, but in the long run this name might be less confusing if we just talk about "id" everywhere.

Reply:
I changed the name to `subEventId`